From a24b6a40ae28997b8543b5912dbcb69be94c3e69 Mon Sep 17 00:00:00 2001
From: Joel Stanley <joel@jms.id.au>
Date: Fri, 11 Sep 2015 14:43:18 +0930
Subject: [PATCH 06/17] Revert "powerpc: Reject binutils 2.24 when building
 little endian"

This reverts commit 60e065f70bdb0b0e916389024922ad40f3270c96.

We know better than Michael does; we carry a patch that fixes our
binutils.

Signed-off-by: Joel Stanley <joel@jms.id.au>
---
 arch/powerpc/Makefile | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/arch/powerpc/Makefile b/arch/powerpc/Makefile
index 05f464e..b2ae85d 100644
--- a/arch/powerpc/Makefile
+++ b/arch/powerpc/Makefile
@@ -358,14 +358,6 @@ checkbin:
 		echo 'disable kernel modules' ; \
 		false ; \
 	fi
-	@if test "x${CONFIG_CPU_LITTLE_ENDIAN}" = "xy" \
-	    && $(LD) --version | head -1 | grep ' 2\.24$$' >/dev/null ; then \
-		echo -n '*** binutils 2.24 miscompiles weak symbols ' ; \
-		echo 'in some circumstances.' ; \
-		echo -n '*** Please use a different binutils version.' ; \
-		false ; \
-	fi
-
 
 CLEAN_FILES += $(TOUT)
 
-- 
2.5.0

